mining/mining.go

```diff
@@ -55,6 +55,13 @@ type TxDesc struct {
 
 	// Fee is the total fee the transaction associated with the entry pays.
 	Fee int64
+
+	// FeePerKB is the fee the transaction pays in Satoshi per 1000 bytes.
+	FeePerKB int64
+
+	// SerializeSize is the cached number of bytes of the underlying
+	// transaction.
+	SerializeSize int
```

Comment:
I would prefer this not be saved.  It is super cheap to compute and, unlike `FeePerKB`, it's not needed anywhere in a hot path.  This would add 8 bytes per transaction which, in turn, reduces the number that can fit into a given amount of memory.  I don't believe the trade off is worth it.